src/rpc/rawtransaction.cpp

```diff
@@ -213,6 +214,142 @@ static UniValue getrawtransaction(const JSONRPCRequest& request)
     return result;
 }
 
+static UniValue searchrawtransactions(const JSONRPCRequest& request) {
```

Comment:
There doesn't seem to be a way to skip transactions anymore. Was this removed intentionally?